steps/web-translate-it/1.0.0/step.yml

```diff
@@ -0,0 +1,38 @@
+title: Web Translate It
+summary: Install web translate it and perform a pull operation.
+description: This step runs a `wti pull` after installing the web translate it gem.
+  You need an account on https://webtranslateit.com to be able to run this step.
+website: https://github.com/mAu888/steps-wti
+source_code_url: https://github.com/mAu888/steps-wti
+support_url: https://github.com/mAu888/steps-wti/issues
+published_at: 2016-05-04T15:25:24.829461663+02:00
+source:
+  git: https://github.com/mAu888/steps-wti.git
+  commit: 7eaefb27968e6a9b5f1453b51f5c9a719172688f
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+type_tags:
+- translation
+- localization
+- xcode
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- WTI_API_KEY: null
+  opts:
+    description: If no `.wti` file can be located in the $WTI_WORKING_DIRECTORY (defaults
+      to current working directory), the step creates a `.wti` file based on the $WTI_API_KEY
+      you provide. If neither a `.wti` file nor an $WTI_API_KEY are given, the step
+      fails.
+    is_required: false
```

Comment:
Yes, the file can be checked into the repository itself. That's how we do it at the moment. You can have some hooks in that `.wti` file. Only if you don't check in the file the step autocreates one by using the API key.